api/tests/validate_legacy_soap_api/validate_soap_endpoints.py

```diff
@@ -0,0 +1,183 @@
+import io
+import logging
+import os
+import re
+import tempfile
+import zipfile
+
+import click
+import requests
+from lxml import etree
+
+import src.adapters.db as db
+import src.logging
+import tests.src.db.models.factories as factories
+from src.adapters.db import PostgresDBClient
+from src.legacy_soap_api.legacy_soap_api_config import SimplerSoapAPI
+from src.legacy_soap_api.legacy_soap_api_schemas import SOAPRequest, SOAPResponse
+from src.legacy_soap_api.legacy_soap_api_utils import get_alternate_proxy_response
+from src.legacy_soap_api.simpler_soap_api import get_simpler_soap_response
+from src.util.local import error_if_not_local
+
+logger = logging.getLogger(__name__)
+
+
+"""
+Add following to soap-api.env
+CERT_DATA_ENCODED =
+KEY_DATA_ENCODED =
+SOAP_URI =
+For cert and key use awk command to replace newlines
+`awk 'NF {sub(/\r/, ""); printf "%s\\n",$0;} bps_grantors.crt`
+`awk 'NF {sub(/\r/, ""); printf "%s\\n",$0;} bps_grantors.key`
+May need to run `poetry install` to update command
+To run use: `docker compose run --rm grants-api poetry run validate-simpler-endpoints`
```

Comment:
Could you add a line to the makefile like

```
validate-simpler-endpoints:
      $(PY_RUN_CMD) validate-simpler-endpoints
```

The `PY_RUN_CMD` resolves to either the above docker-compose command OR if you've setup your local development to work outside of Docker (complex, but way way faster) then it ends up being just the `poetry run` bit.